rfcs/0114-Abuse protection for RabbitMQ server and tc-events.md

```diff
@@ -0,0 +1,28 @@
+# RFC 114 - Abuse protection for RabbitMQ server and tc-events.md
+* Comments: [#114](https://api.github.com/repos/taskcluster/taskcluster-rfcs/issues/114)
+* Proposed by: @Biboswan
+
+# Summary
+
+1: limit connection attempt rate to tc-events(frequent opening and closing connection => higher CPU utilization)
```

Comment:
Is this per source IP?  How would we handle false-positives from shared IPs?  For example, everyone in the Mozilla Mountain View office will come to this service from the same IP..